wp-cli-templates/block-index-js.mustache

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ExternalLink, Path } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import renderMaterialIcon from '../../utils/render-material-icon';
+{{#externalEdit}}
+import edit from './edit';
+{{/externalEdit}}
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name     = '{{ slug }}';
+export const title    = __( '{{ title }}', 'jetpack' );
+export const settings = {
+	title,
+
+	description: (
+		<Fragment>
+			<p>{ __( '{{ description }}', 'jetpack' ) }</p>
+			<ExternalLink href="#">{ __( 'Learn more about {{ title }}', 'jetpack' ) }</ExternalLink>
+		</Fragment>
+	),
+
+	/* @TODO add icon */
```

Comment:
We could give folks a head-start by giving them this URL here: https://material.io/tools/icons/?style=outline

Reply:
Done